tests/test_bnn.py

```diff
@@ -1,4 +1,5 @@
 from functools import partial
+from packaging import version
```

Comment:
Is this still needed somewhere?

Reply:
Ah sorry, misread the end of the file. But as per my earlier comment, I'd remove this and assume pyro version 1.8.1.